scripts/error_codes.py

```diff
@@ -172,26 +170,29 @@ def examine_id_coverage(top_dir, used_ids):
 
 
 def main(argv):
-    # pylint: disable=too-many-branches, too-many-locals
+    # pylint: disable=too-many-branches, too-many-locals, too-many-statements
 
     check = False
     fix = False
-    noconfirm = False
+    no_confirm = False
     examine_coverage = False
-    opts, args = getopt.getopt(argv, "", ["check", "fix", "noconfirm", "examine-coverage"])
+    next = False
+    opts, args = getopt.getopt(argv, "", ["check", "fix", "no-confirm", "examine-coverage", "next"])
 
     for opt, arg in opts:
         if opt == '--check':
             check = True
         elif opt == "--fix":
             fix = True
-        elif opt == '--noconfirm':
-            noconfirm = True
+        elif opt == '--no-confirm':
+            no_confirm = True
         elif opt == '--examine-coverage':
             examine_coverage = True
+        elif opt == '--next':
+            next = True
 
-    if not check and not fix and not examine_coverage:
-        print("usage: python error_codes.py --check | --fix [--noconfirm] | --examine-coverage")
+    if not check and not fix and not examine_coverage and not next:
```

Comment:
```suggestion
    if True not in [check, fix, examine_coverage, next]:
```

Reply:
Would 
```
if [check, fix, examine_coverage, next].count(True) == 1:
```
be better? The flags are not supposed to be combined.

Reply:
Yes. That works